api/src/main/java/uk/gov/dvsa/mot/persist/jdbc/MotTestReadDaoJdbc.java

```diff
@@ -74,82 +69,29 @@ public void setConnectionManager(ConnectionManager connectionManager) {
     @Override
     public MotTest getMotTestById(long id) {
 
-        logger.debug("Entry getMotTestById " + id);
-        if (motTestMap.containsKey(id)) {
-            logger.debug("Exit getMotTestById Returning Cached MotTest id " + id);
-            return motTestMap.get(id);
-        } else {
-            MotTest motTest = getMotTestCurrentById(id);
+        MotTest motTest = getMotTestCurrentById(id);
```

Comment:
I see a lot of debug statements removed. Do you think they are unnecessary? (I would have probably changed them to trace or kept them as is.)